Started implementing w/o PID

ezhang887 opened the pull request:
Need to test this code before proceeding

ezhang887 committed:
Started implementing w/o PID

Need to test this code before proceeding

cwyant reviewed (changes requested):
Good start! Can you put in a summary when you create a pull request how you plan to test (in detail) or how you have verified its working?

ezhang887 committed:
Greatly improved accuracy of distance to drive

This implements suggestions suggested earlier (e.g. adding additional
parameters to the constructor) as well as greatly improves the accuracy
of the movement (at least at 0.25 power - still need to test other
powers)

ezhang887 committed:
Further progression

Improved accuracy of MoveForward (with power level 0.4), tuned Turn, and
started autonomous integration

cwyant reviewed (approved):
Looks good, go ahead and push this in
